invenio_rdm_records/fixtures/demo.py

```diff
@@ -34,12 +36,14 @@ class CachedVocabularies:
     def fake_resource_type(cls):
         """Generate a random resource_type."""
         if not cls._resource_type_ids:
-            results = vocabulary_service.search(
-                system_identity, type='resource_types'
-            )
-            cls._resource_type_ids = [
-                r["id"] for r in results.to_dict()["hits"]["hits"]
-            ]
+
+            dir_ = Path(__file__).parent
+
+            cls._resource_type_ids = list(VocabulariesFixture(
+                system_identity,
+                [Path("./app_data"), dir_ / "data"],
+                "vocabularies.yaml",
+            ).get_records_by_vocabulary("resource_types"))
 
         random_id = random.choice(cls._resource_type_ids)
```

Comment:
This will return a full record but here we only need `random_id["id"]`?

Reply:
Thanks! Fixed! :)

Reply:
I will remove the `print` in a minute